config/resource-manager.hcl

```
@@ -67,7 +67,7 @@ service "blueprint" {
 }
 service "chaos" {
   name      = "ChaosStudio"
-  available = ["2022-07-01-preview"]
+  available = ["2022-07-01-preview", "2023-04-15-preview"]
```

Can you please remove the older version since it isn't being use anywhere yet, we're trying to keep an eye on getting rid of unused versions since it currently impact the performance of the data API.
```suggestion
  available = ["2023-04-15-preview"]
```

---

Fixed.